rdb/Connection.php

```diff
@@ -29,7 +29,8 @@ public function __construct(
         $port = null,
         $db = null,
         $apiKey = null,
-        $timeout = null
+        $timeout = null,
+        $connect = false
```

Comment:
The default should be `true`.
Not connecting when calling `connect()` is a big change, since it completely changes when connection errors are thrown, as well as the timing of applications.